official/transformer/v2/misc.py

```diff
@@ -68,7 +68,8 @@ def define_transformer_flags():
       intra_op=False,
       synthetic_data=True,
       max_train_steps=False,
-      dtype=False,
+      dtype=True,
+      loss_scale=True,
```

Comment:
why true by default?

Reply:
"True" means to have the --dtype and --loss_scale flags, which I am now exposing.